test.js

```diff
@@ -1,9 +1,29 @@
-module.exports = {
-  extends: './eslint.js',
+const eslint = require('./eslint.js');
```

Comment:
Such naming may be confusing

Reply:
Done. 
https://github.com/hexojs/eslint-config-hexo/pull/64/commits/0544bddb1bcf813bff3ba3dfaf4c7b357dd280bd

Personally, I think it would be better to rename the file itself from `eslint.js`. If we were to align it with other file names, it would be `js.js`, but I don't think that’s a good choice for naming.